common/build/eslint-config-fluid/strict-ts.json

```diff
@@ -0,0 +1,1440 @@
+    ],
+    "unicorn/prefer-array-flat": [
+      "error"
+    ],
+    "unicorn/prefer-array-flat-map": [
+      "error"
+    ],
+    "unicorn/prefer-array-index-of": [
+      "error"
+    ],
+    "unicorn/prefer-array-some": [
+      "error"
+    ],
+    "unicorn/prefer-at": [
+      "off"
+    ],
+    "unicorn/prefer-code-point": [
+      "error"
+    ],
+    "unicorn/prefer-date-now": [
+      "error"
+    ],
+    "unicorn/prefer-default-parameters": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-append": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-dataset": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-remove": [
+      "error"
+    ],
+    "unicorn/prefer-dom-node-text-content": [
+      "error"
+    ],
+    "unicorn/prefer-export-from": [
+      "error"
+    ],
+    "unicorn/prefer-includes": [
+      "error"
+    ],
+    "unicorn/prefer-json-parse-buffer": [
+      "error"
+    ],
+    "unicorn/prefer-keyboard-event-key": [
+      "error"
+    ],
+    "unicorn/prefer-math-trunc": [
+      "error"
+    ],
+    "unicorn/prefer-modern-dom-apis": [
+      "error"
+    ],
+    "unicorn/prefer-module": [
+      "error"
+    ],
+    "unicorn/prefer-negative-index": [
+      "error"
+    ],
+    "unicorn/prefer-node-protocol": [
+      "error"
+    ],
+    "unicorn/prefer-number-properties": [
+      "error"
+    ],
+    "unicorn/prefer-object-from-entries": [
+      "error"
+    ],
+    "unicorn/prefer-optional-catch-binding": [
+      "error"
+    ],
+    "unicorn/prefer-prototype-methods": [
+      "error"
+    ],
+    "unicorn/prefer-query-selector": [
+      "error"
+    ],
+    "unicorn/prefer-reflect-apply": [
+      "error"
+    ],
+    "unicorn/prefer-regexp-test": [
+      "error"
+    ],
+    "unicorn/prefer-set-has": [
+      "error"
+    ],
+    "unicorn/prefer-spread": [
+      "error"
+    ],
+    "unicorn/prefer-string-replace-all": [
+      "off"
+    ],
+    "unicorn/prefer-string-slice": [
+      "error"
+    ],
+    "unicorn/prefer-string-starts-ends-with": [
+      "error"
+    ],
+    "unicorn/prefer-string-trim-start-end": [
+      "error"
+    ],
+    "unicorn/prefer-switch": [
+      "error"
+    ],
+    "unicorn/prefer-ternary": [
+      "error"
+    ],
+    "unicorn/prefer-top-level-await": [
+      "off"
+    ],
+    "unicorn/prefer-type-error": [
+      "error"
+    ],
+    "unicorn/relative-url-style": [
+      "error"
+    ],
+    "unicorn/require-array-join-separator": [
+      "error"
+    ],
+    "unicorn/require-number-to-fixed-digits-argument": [
+      "error"
+    ],
+    "unicorn/require-post-message-target-origin": [
+      "off"
+    ],
+    "unicorn/string-content": [
+      "off"
+    ],
+    "unicorn/template-indent": [
+      "warn"
+    ],
+    "unicorn/throw-new-error": [
+      "error"
+    ],
+    "dot-notation": [
+      "off",
+      {
+        "allowPattern": "^_",
+        "allowKeywords": true
+      }
+    ],
+    "no-unused-expressions": [
+      "off"
+    ],
```

Comment:
Make a warning.

Reply:
May be redundant with tsc enforcement.

Reply:
I may be in the minority, but I would rather unused expressions be a lint-time error than a compilation error.